test/kata4.humanCatDogYears.test.js

```diff
@@ -1,3 +1,7 @@
 const { humanCatDogYears } = require("../src");
 
-// Look Ma, no handlebars!!!
+describe("humanCatDogYears", () => {
+  test("returns array of human, cat and dog years when passed human years", () => {
+    expect(humanCatDogYears(10)).toEqual([10, 56, 64]);
```

Comment:
We should aim to write at least 3 assertions per function, when testing it. Like, if number is 1, what are we expecting?